.circleci/config.yml

```diff
@@ -102,7 +102,7 @@ workflows:
           context: determined-production
           matrix:
             parameters:
-              image-type: ["tf1-cpu", "tf2-cpu", "tf1-gpu", "cuda-11"]
+              image-type: ["tf1-cpu", "tf2-cpu", "tf1-gpu", "cuda-11", "tf25-gpu"]
```

Comment:
Now that it's the default, how about we start calling the cuda-11 image the -tf2 image? I could do that in a separate change. There's no point in separate CUDA 11 tests anymore either.

Reply:
I agree, `tf2-gpu`.